zgrab_schema.py

```diff
@@ -337,11 +337,16 @@
     "headers":zgrab_http_headers
 })
 
+zgrab_http_request_response = SubRecord({
+    "request":zgrab_http_request,
+    "response":zgrab_http_response
+})
+
 zgrab_http = Record({
     "data":SubRecord({
       "http":SubRecord({
-        "request":zgrab_http_request,
-        "response":zgrab_http_response
+        "response":zgrab_http_response,
+        "request_response_chain":ListOf(zgrab_http_request_response)
```

Comment:
Also defined in this file there's a `zgrab_http_proxy` object (or something similarly named). Can you make sure that it is still to date?